### PR TITLE
Negation matcher fix

### PR DESCRIPTION
**Pull request by maksimf**

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -1,3 +1,6 @@
+3.2.6
```
Comment (patricioe) [inline CHANGES.txt]: Please rebase. We are still a 3.2.4-rc3

File: lib/splitclient-rb/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module SplitIoClient
-  VERSION = '3.2.4-rc2'
+  VERSION = '3.2.6'
```
Comment (patricioe) [inline lib/splitclient-rb/version.rb]: discard this change please.

File: lib/engine/parser/condition.rb [hunk 3]
```diff
@@ -38,20 +36,24 @@ def matchers
       @data[:matcherGroup][:matchers]
     end
 
-    def matcher_all_keys params
+    def negation_matcher(matcher)
+      NegationMatcher.new(matcher)
+    end
+
+    def matcher_all_keys(_params)
       AllKeysMatcher.new
```
Comment (patricioe) [inline lib/engine/parser/condition.rb]: can we reuse the object?

Comment (maksimf) [inline lib/engine/parser/condition.rb]: @patricioe for sure, done!

**Pull request merged**